src/features/auth/Login.tsx

```diff
@@ -5,7 +5,7 @@ import { FunctionComponent, useCallback, useState } from "react";
 import loginService from "features/auth/services/login.service";
 import { useDispatch } from "react-redux";
 import { login as loginAction } from "features/auth/store";
-import useRedirectWhenLogged from "./use-redirect-when-logged";
+import useRedirectWhenLogged from "../../hooks/use-redirect-when-logged";
```

Comment:
Si es una importación de tan atrás, hazlo una importación absoluta
```suggestion
import useRedirectWhenLogged from "hooks/use-redirect-when-logged";
```

Reply:
Chevere, lo que pasa es que eso fue antes de que agregaramos el tema de la importacion absoluta xD

Reply:
Claro po, pero ahora que lo agregaste hay que hacerlo xP